evalml/pipelines/components/estimators/regressors/decision_tree_regressor.py

```diff
@@ -0,0 +1,39 @@
+from sklearn.tree import DecisionTreeRegressor as SKDecisionTreeRegressor
+from skopt.space import Integer
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+
+
+class DecisionTreeRegressor(Estimator):
```

Comment:
If you don't mind, I'll file an issue for exposing the `tree_` property of these estimators. Looks like sklearn has cool features to visualize these models that we could leverage in our `model_understanding` module.

Reply:
Yes!! Please do. I saw this and was really tempted to just add it to this PR but filing a separate issue is definitely a better idea 😁

Reply:
Filed #1239!

Reply:
Thanks!!